package.json

```diff
@@ -35,19 +35,42 @@
     "networks"
   ],
   "dependencies": {
-    "chalk": "^1.1.3",
+    "auto-bind": "^1.1.0",
     "clipboardy": "^0.1.1",
     "conf": "^1.0.0",
     "got": "^6.3.0",
     "has-ansi": "^2.0.0",
+    "import-jsx": "^1.0.0",
+    "ink": "github:vadimdemedes/ink",
     "lodash.debounce": "^4.0.6",
-    "log-update": "^1.0.2",
     "mem": "^1.1.0",
     "meow": "^3.7.0",
     "skin-tone": "^1.0.0"
   },
   "devDependencies": {
     "ava": "*",
-    "xo": "^0.16.0"
+    "eslint-config-xo-react": "^0.12.0",
+    "eslint-plugin-react": "^7.1.0",
+    "xo": "^0.18.2"
+  },
+  "xo": {
+    "extends": [
+      "xo-react"
+    ],
+    "plugins": [
+      "react"
+    ],
+    "rules": {
+      "react/prop-types": 0,
+      "react/jsx-closing-bracket-location": [
+        "error",
+        "tag-aligned"
+      ]
```

Comment:
I changed this, because with this setting JSX looks better and it seems to be a standard:

Before:

```jsx
<Component
  a={1}
  b={2}
  />
  <Child/>
</Component>
```

After:

```jsx
<Component
  a={1}
  b={2}
/>
  <Child/>
</Component>
```

I changed this to suit my eye during dev, but forgot to change it back. If you prefer the first one, I can rollback ;)